frontend/projects/ui/src/app/pages/apps-routes/app-show/components/app-show-additional/app-show-additional.component.html

```diff
@@ -0,0 +1,71 @@
+<ion-item-divider>Additional Info</ion-item-divider>
```

Comment:
is there a way to reuse this html file between both the marketplace and app show additional components?

Reply:
they're not thaaaat [different](https://www.diffchecker.com/EqHM95Q6) (just saying), but I understand